presto-native-execution/presto_cpp/main/TaskManager.h

```diff
@@ -107,6 +111,10 @@ class TaskManager {
     return &queryContextManager_;
   }
 
+  inline const std::shared_ptr<velox::exec::PartitionedOutputBufferManager> getBufferManager() const {
```

Comment:
naming: getBufferManager -> bufferManager

Use const & for the return type or return raw pointer.